platform/viewer/src/appExtensions/MeasurementsPanel/init.js

```diff
@@ -3,6 +3,9 @@ import cornerstone from 'cornerstone-core';
 import csTools from 'cornerstone-tools';
 import throttle from 'lodash.throttle';
 
+import LabellingFlow from '../../components/Labelling/LabellingFlow';
+import ToolContextMenu from '../../connectedComponents/ToolContextMenu';
```

Comment:
Ah. Yes.
After we've completely decoupled the `MeasurementsExtension`, we'll want to pull it into it's own project, so these need to live closer.

Pretend `appExtensions/MeasurementsPanel` is a boundary we shouldn't cross for imports, unless using something like `@ohif/ui`